pkg/transform/report_output.go

```diff
@@ -13,6 +14,14 @@ type ReportOutput struct {
 	ComponentReports []ComponentReport `json:"components"`
 }
 
+// NodeResources represents a json report of Node resources
+type NodeResources struct {
+	CPU            string `json:"cpu"`
+	MemoryConsumed string `json:"memoryConsumed"`
+	MemoryCapacity string `json:"memoryCapacity"`
+	RunningPods    string `json:"runningPods"`
```

Comment:
This place is mentioned above. I didn't check the rest fields, those may have the similar issue.

Reply:
In addition to the previous explanation: The structure does not have a straight analogy in k8s api, so we are free to fantasize here.

But I agree that there is a room for improvement here. Going to look what to change.